src/main/resources/assets/tiefix/lang/zh_cn.json

```diff
@@ -0,0 +1,43 @@
+{
+  "options.tiefix.title": "TieFix 设置",
+  "options.tiefix.category.ui": "UI & HUD",
+  "options.tiefix.category.controls": "控制",
+  "options.tiefix.category.audioVisual": "音频与视觉",
+  "options.tiefix.category.gameplay": "游戏性和运动",
+  "options.tiefix.category.misc": "杂项",
+  "options.tiefix.mc2071": "允许在下界传送门中打开 GUI",
+  "options.tiefix.mc2071.bug": "MC-2071",
+  "options.tiefix.mc4490": "修复第三人称下的钓鱼线,
```

Comment:
Looks like you are missing a closing quote here.

Reply:
> Looks like you are missing a closing quote here.

Done.Sorry for the mistake. :(